src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -259,4 +264,24 @@ public void onHide() {
     // Hide all games to free up memory
     filteredItems.setPredicate(game -> false);
   }
+
+  public void onMapsManagementButtonClicked() {
+
+    // TODO: Move button to another space in future
```

Comment:
I think this would be best placed in the map vault. There is a button used to manage mods in the mod vault which is hidden in the map vault